Bot/BotInstance.cs

```diff
@@ -829,7 +840,7 @@ public async Task<DateTime> DoBattleAsync()
                 {
                     CurrentlyActive = false;
                 }
-                Settings.LogSummaryList.Add((LogSummary.Index, LogSummary.Account, LogSummary.BattleResult, LogSummary.Rating, LogSummary.ECR, LogSummary.QuestStatus));
+                Settings.LogSummaryList.Add((LogSummary.Index, LogSummary.Account, LogSummary.BattleResult, LogSummary.Rating, LogSummary.ECR, LogSummary.QuestStatus,LogSummary.SPSStake));
```

Comment:
i would suggest a space between the comma and LogSummary.SPSStake for readability and to match style